native/src/routes/SearchModal.tsx

```diff
@@ -113,6 +114,7 @@ const SearchModal = ({
         <List
           items={searchResults}
           renderItem={renderItem}
+          accessibilityLabel={`${searchCategories(categories, query).length} ${t('searchResults')}`}
```

Comment:
This should go to translations.json with proper plural handling and using a placeholder for the number.